apicast/src/oauth/keycloak.lua

```diff
@@ -248,7 +262,8 @@ function _M:get_token(service, client)
   -- call Keycloak authorize
   local url = self.config.token_url
 
-  local res = http_client.post(url, req_body)
+  local headers = ngx.req.get_headers()
+  local res = http_client.post(url, ngx.req.get_post_args(), { headers = headers })
```

Comment:
I think one positive test will be fine (checking that the Authorization is passed) and if we have to stub some method that returns a table then we can add some extra headers there and see that they are not in keycloak.